arches/app/models/tile.py

```diff
@@ -545,8 +545,8 @@ def delete(self, *args, **kwargs):
                     datatype = self.datatype_factory.get_instance(node.datatype)
                     datatype.post_tile_delete(self, nodeid, index=index)
 
-                    resource = Resource.objects.get(pk=self.resourceinstance_id)
-                    resource.save_descriptors()
+                resource = Resource.objects.get(pk=self.resourceinstance_id)
+                resource.save_descriptors()
```

Comment:
I think we could skip the call to save_descriptors if the tile being deleted has no data.

Reply:
I looked into this and it turns out that the descriptor can persist if a semantic parent tile of a descriptor tile is deleted, so we do need to call save_descriptors